csrankings-9.csv

```diff
@@ -700,6 +700,7 @@ Wenliang Du,Syracuse University,http://www.cis.syr.edu/~wedu,1aMkcasAAAAJ
 Wenpin Jiao,Peking University,http://sei.pku.edu.cn/~jwp,NOSCHOLARPAGE
 Wenping Wang,University of Hong Kong,https://i.cs.hku.hk/~wenping,28shvv0AAAAJ
 Wenqing Zhang,Fudan University,http://homepage.fudan.edu.cn/wqzhang/testete,NOSCHOLARPAGE
+Wenrui Chen,Hunan University,http://eeit.hnu.edu.cn/info/1318/4594.htm,c3bvnWIAAAAJ

```

Comment:
Not in CS; not clearly eligible for inclusion.

Reply:
Yes. He is in ECE department but can advise Phd students with (bachelor) background in Computer Science and he is doing Robotics. Not so sure if that is eligible.